packages/cli/src/cli/provider/provider.util.ts

```diff
@@ -0,0 +1,103 @@
+import { TileMetadataProviderRecord } from '@basemaps/lambda-shared';
+import * as c from 'ansi-colors';
+import { inspect } from 'util';
+
+export function printProvider(data: TileMetadataProviderRecord, changes = data): void {
+    console.log(c.bold('Title:'), data.serviceProvider.name);
+    console.log(c.bold('CreatedAt:'), new Date(data.createdAt).toISOString());
+    console.log(c.bold('UpdatedAt:'), new Date(data.updatedAt).toISOString());
+    console.log(c.bold('Version:'), `v${data.version}`);
+    console.log(
+        c.bold('Content:') +
+            inspect(
+                {
+                    serviceProvider: changes.serviceProvider,
+                    serviceIdentification: changes.serviceIdentification,
+                },
+                false,
+                5,
+                true,
+            ),
+    );
+}
+
+export const BlankProvider: TileMetadataProviderRecord = {
+    createdAt: 0,
+    id: '',
+    updatedAt: 0,
+    version: 0,
+    revisions: 0,
+    serviceIdentification: {
+        accessConstraints: '',
+        description: '',
+        fees: '',
+        title: '',
+    },
+    serviceProvider: {
+        contact: {
+            address: {
+                city: '',
+                country: '',
+                deliveryPoint: '',
+                email: '',
+                postalCode: '',
+            },
+            individualName: '',
+            phone: '',
+            position: '',
+        },
+        name: '',
+        site: '',
+    },
+};
+
+export function validateProvider(after: Record<string, any>, before: Record<string, any>): any {
```

Comment:
Needs some comments to describe what this actually does?

also why does it return a any?